sys-libs/libnbd/libnbd-1.14.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit autotools bash-completion-r1 python-single-r1
+
+DESCRIPTION="NBD client library in userspace"
+HOMEPAGE="https://gitlab.com/nbdkit/libnbd"
+SRC_URI="https://download.libguestfs.org/libnbd/$(ver_cut 1-2)-stable/${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples fuse gnutls +uri-support go ocaml python test"
+
+RESTRICT="!test? ( test ) test? ( userpriv )"
+
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+"
+RDEPEND="
+	examples? ( dev-libs/glib dev-libs/libev )
+	fuse? ( sys-fs/fuse:3 )
+	gnutls? ( net-libs/gnutls:= )
+	go? ( dev-lang/go )
+	ocaml? ( >=dev-lang/ocaml-4.03:=[ocamlopt] )
+	python? ( ${PYTHON_DEPS} )
+	uri-support? ( dev-libs/libxml2 )
```

Comment:
libxml2 is an extremely common dep. If you say things often need uri-support in libnbd, let's just always enable it.

Reply:
fix
require libxml by default and I also had it activated but it was because I did not change many things in the ebuild, but I also see it as a good thing that it is forced